src/shell/meanbee/configpoweredcss/regenerate.php

```diff
@@ -0,0 +1,31 @@
+<?php
+require_once __DIR__ . '/../../abstract.php';
+
+class Meanbee_Configpoweredcss_Regenerate extends Mage_Shell_Abstract
+{
+
+    /**
+     * Run script
+     *
+     */
+    public function run()
+    {
+        /**
+         * Shell script regenerate the configpowered css for all stores
+         */
+
+        try {
+            Mage::helper('meanbee_configpoweredcss')->regenerateConfigPoweredCss();
+        } catch (Exception $e) {
+            fwrite(STDERR, sprintf('There was an error when regenerating the config powered css: %s',
+                $e->getTraceAsString()));
```

Comment:
Like Tom mentioned before, the convention is either to keep the entire method on one line, or split it to one argument per line, to ease readability.

Reply:
Damn, my linter changed it and I didn't notice it.